build.sbt

```diff
@@ -139,14 +136,11 @@ lazy val genInput =
 
 lazy val genOutput = projectMatrix
   .in(file("gen/output"))
+  .enablePlugins(NoPublishPlugin)
   .settings(
-    publish / skip := true,
-    scalacOptions ++=
-      (scalaVersion.value match {
-        case `scala3Version` => Nil
-        case _               => List("-Wconf:cat=unused:info")
-      }),
-    libraryDependencies ++= Settings.Libraries.Monocle.value
+    scalacOptions ++= { if (tlIsScala3.value) Nil else List("-Wconf:cat=unused:info") },
+    libraryDependencies ++= Settings.Libraries.Monocle.value,
+    tlFatalWarnings := false
```

Comment:
This reminded me that we are in all projects disabling fatal warnings on dev but enabling them in CI to ease development. Is that still the case?

Reply:
Yes:
https://github.com/gemini-hlsw/sbt-lucuma/blob/3ce8fc26c6e01cd4931b7f0a8ecbf678a567ec32/core/src/main/scala/lucuma/sbtplugin/LucumaPlugin.scala#L71